columnflow/tasks/histograms.py

```diff
@@ -199,9 +199,10 @@ def expr(events, *args, **kwargs):
                             return route.apply(events, null_value=variable_inst.null_value)
                     # apply it
                     fill_kwargs[variable_inst.name] = expr(events)
+
                 # broadcast and fill
```

Comment:
Could you adjust the comment maybe? :)

Reply:
Ok, after looking at the [docs](https://awkward-array.org/doc/main/reference/generated/ak.cartesian.html), "broadcast" still makes sense here.